skel/handlers/index.go

```diff
@@ -31,6 +32,11 @@ func Setup(app *sonic.Runtime) {
 	router.GET("/version", version)
 
 	SetPageRoutes(router)
+
+	opx := router.Group("/api/v2/vault")
```

Comment:
I think `/api/v2/config` would be better
We'd need to have endpoints for public(env.js) and all scopes(tower)

Reply:
I saw we move to /admin. I will leave it as it is.